packages/editor/src/client/routes/comments/commentEditView.tsx

```diff
@@ -27,11 +25,6 @@ const showErrors = (error: ApolloError): void => {
   )
 }
 
-const richTextBlock = BlockMap[BlockType.RichText]
```

Comment:
Not needed anymore as the RichTextBlock now sets the default if the  text is empty